shuup/admin/modules/orders/action_items.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
```

Comment:
maybe consider creating a `toolbar.py` instead of `action_items.py` to follow suit with products.